fdbclient/FileBackupAgent.actor.cpp

```diff
@@ -3393,6 +3390,23 @@ struct StartFullBackupTaskFunc : BackupTaskFuncBase {
 			// the flag was not set before.
 			wait(success(ManagementAPI::changeConfig(cx.getReference(), "backup_worker_enabled:=1", true)));
 			backupWorkerEnabled = true;
+			// the user is responsible for manually disabling backup worker after the last backup is done
+		}
+
+		// Get start version after backup worker are enabled
```

Comment:
@sfc-gh-huliu could you please advise why do we want to get start version after we change the config? is there a bug if we keep this part as it previously was?